lib/tree/peer.go

```diff
@@ -0,0 +1,74 @@
+package tree
+
+import (
+	"github.com/dedis/cothority/lib/hashid"
+	"github.com/dedis/cothority/lib/network"
+	"github.com/dedis/crypto/abstract"
+	"strconv"
+)
+
+// Peer represents a Cothority-member
+type Peer struct {
+	// The hostname of the peer
+	Name string
+	// The port-number
+	Port int
+	// The public-key
+	PubKey abstract.Point
+	// The private-key - mostly for ourselves
+	PrivKey abstract.Secret
```

Comment:
That's true. I just thought that it's the most convenient place to put it by putting similar things together. Else I'll have to put the private key somewhere in the `Node` structure (which doesn't exist yet)